pkg/generate/app/pipeline.go

```diff
@@ -479,7 +479,8 @@ func (a *acceptBuildConfigs) Accept(from interface{}) bool {
 	if err != nil {
 		return false
 	}
-	return gvk[0].GroupKind() == build.Kind("BuildConfig") || gvk[0].GroupKind() == image.Kind("ImageStream")
+	return gvk[0].GroupKind() == build.Kind("BuildConfig") || gvk[0].GroupKind() == image.Kind("ImageStream") ||
```

Comment:
After looking at several hundred of these, it probably would have been better to have created a common utility method that did simple comparison of GK/GR "in" a set of GK/GR.  I don't have a ton of confidence that every one of these is right, and it's hard to review.